bbb_ajax.php

```diff
@@ -37,6 +37,7 @@
 $params['signed_parameters'] = optional_param('signed_parameters', '', PARAM_TEXT);
 $params['updatecache'] = optional_param('updatecache', 'false', PARAM_TEXT);
 $params['meta'] = optional_param('meta', '', PARAM_TEXT);
+$params['delete'] = optional_param('delete', false, PARAM_BOOL);
```

Comment:
Wht is the new delete used for?

Reply:
The delete is used for deleting the guest password